packages/bot-web-ui/src/app.jsx

```diff
@@ -38,7 +40,15 @@ class App extends React.Component {
         } = this.root_store;
         const { handleFileChange } = saveload;
         const { toggleStrategyModal } = quick_strategy;
-        this.dbot_store = { is_mobile: false, client, flyout, toolbar, toggleStrategyModal, handleFileChange };
+        this.dbot_store = {
+            is_mobile: false,
```

Comment:
You can use `isMobile()` function from the `@deriv/shared/utils/screen` to set the default for `is_mobile`.

Reply:
That file doesn't exist (yet) on latest branch. I'll add a `TODO` to replace this once it's merged.